pkg/cmd/cli/cmd/deploy.go

```diff
@@ -352,6 +361,9 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer)
 
 // reenableTriggers enables all image triggers and then persists config.
 func (o DeployOptions) reenableTriggers(config *deployapi.DeploymentConfig, out io.Writer) error {
+	if config.Spec.Paused {
+		return fmt.Errorf("cannot enable triggers for a paused deployment config")
```

Comment:
We want `paused` to have triggers set to automatic:false. I think we need to enforce it on the server too. When a deployment config is paused then we shouldnt' allow updates on automatic: false to automatic: true or the addition of a config change trigger.